ably-ios/ARTDataQuery.m

```diff
@@ -49,4 +50,16 @@ - (NSMutableArray *)asQueryItems {
 
 @implementation ARTRealtimeHistoryQuery
 
+- (NSMutableArray *)asQueryItems {
+    NSMutableArray *items = [super asQueryItems];
+    if (self.untilAttach) {
+        NSAssert(self.realtimeChannel, @"ARTRealtimeHistoryQuery used from outside ARTRealitmeChannel.history");
+        if (self.realtimeChannel.state != ARTRealtimeChannelAttached) {
+            @throw [NSError errorWithDomain:ARTAblyErrorDomain code:ARTRealtimeHistoryErrorNotAttached userInfo:nil];
+        }
+        [items addObject:[NSURLQueryItem queryItemWithName:@"from_serial" value:self.realtimeChannel.attachSerial]];
```

Comment:
I believe this should be `fromSerial` as we changed all of our query params to use camelCase